ApplicationLibCode/Application/Tools/RiaWellLogUnitTools.inl

```diff
@@ -279,7 +279,7 @@ bool RiaWellLogUnitTools<FloatType>::convertValues( const std::vector<FloatType>
         return true;
     }
     else if ( ( stringsMatch( unitsIn, pascalUnitString() ) ||
```

Comment:
@kriben , this code looks suspicious, line 281-282. Can you comment on the correct logic?